qemu/tests/cfg/test_SMM_enabled.cfg

```diff
@@ -0,0 +1,8 @@
+- test_SMM_enabled:
+    type = test_SMM_enabled
+    no Host_RHEL.6 Host_RHEL.7 Host_RHEL.8.u0 Host_RHEL.m8.u1 Host_RHEL.m8.u2 Host_RHEL.m8.u3 Host_RHEL.m8.u4 Host_RHEL.m8.u5 Host_RHEL.m8.u6
+    vms = ""
+    auto_cpu_policy = "virttest"
+    source_file = 'smm_enabled/smm_int_window.flat'
+    error_msg = "hardware error 0x80000021"
+    test_cmd = "--no-reboot -nodefaults -device pc-testdev -device isa-debug-exit,iobase=0xf4,iosize=0x4 -vnc none -serial stdio -device pci-testdev -machine accel=kvm -kernel %s -smp 2 -machine smm=on -cpu host -m 2g -M q35"
```

Comment:
Not a good idea, qemu_vm also has a method to add the kernel file.
```
        def add_kernel(filename):
            return " -kernel '%s'" % filename
```

Reply:
> add_kernel

But this script is not a regular way to create qemu command by function `def make_create_command(self, name=None, params=None, root_dir=None)`.

Reply:
> Not a good idea, qemu_vm also has a method to add the kernel file.
> 
> ```
>         def add_kernel(filename):
>             return " -kernel '%s'" % filename
> ```

update and thanks.